src/main/java/fi/metatavu/metaform/server/metaforms/ReplyController.java

```diff
@@ -313,7 +366,180 @@ public void deleteReply(Reply reply) {
   public List<ReplyField> listReplyFields(Reply reply) {
     return anyReplyFieldDAO.listByReply(reply);
   }
+  
+  /**
+   * Returns table field as valid table data. 
+   * 
+   * If input object is not valid null is returned instead
+   * 
+   * @param columnMap column map
+   * @param value input object
+   * @return table data
+   */
+  @SuppressWarnings("unchecked")
+  private List<Map<String, Object>> getTableValue(Map<String, MetaformTableColumn> columnMap, Object value) {
+    if (value == null) {
+      return Collections.emptyList();
+    }
+    
+    if (!(value instanceof List)) {
+      return null;
+    }
+    
+    List<Object> listValue = (List<Object>) value;
+    if (listValue.isEmpty()) {
+      return Collections.emptyList();
+    }
+    
+    for (Object listItem : listValue) {
+      if (!(listItem instanceof Map)) {
+        return null;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Return an empty collection instead of null. ([squid:S1168](https://sonarcloud.io/organizations/metatavu-oy/rules#rule_key=squid%3AS1168))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=fi.metatavu.metaform-api%3Ametaform-api&pullRequest=60&issues=AWY4b1pzvOWp7Yd6rqjm&open=AWY4b1pzvOWp7Yd6rqjm)

[ISSUE_KEY]: AWY4b1pzvOWp7Yd6rqjm